plugins/jquery-filters.php

```diff
@@ -35,6 +35,16 @@
 }
 unset( $sites, $options, $option );
 
+// Ensure that the local port is used for template assets, if it exists.
+add_filter( 'theme_root_uri', function( $value ) {
+	if ( JQUERY_STAGING === 'local' ) {
+		// Don't specify http vs https here, as the site may be accessed via either.
+		$siteurl = '//' . strtr( JQUERY_STAGING_FORMAT, [ '%s' => JQUERY_LIVE_SITE ] );
```

Comment:
Can we have it documented somewhere what `JQUERY_STAGING` and `JQUERY_STAGING_FORMAT` are supposed to be in which case? At least I didn't see anything in this repo.

Reply:
Sure. These are set in the environment by jquery-wp-docker locally or in the deployed environments.

Reply:
@timmywil Can you also describe `JQUERY_STAGING_FORMAT`?

Reply:
Where is this defined? Searching across the org doesn't show me any comments in generated files: https://github.com/search?q=org%3Ajquery%20JQUERY_STAGING_FORMAT&type=code

Reply:
Ah, you didn't mean comments but the value which indicates how it should be used...

I guess for our purposes it may be fine, but do we consider `jquery-wp-docker` the only supported way to run the sites? I always thought the answer was no, so options should be described in `jquery-wp-contents`.

Reply:
Instructions for running the site without docker have been neglected. It may be possible, but (1) I don't know how to do it (2) I don't know where those instructions are (3) I'm not sure we need to support it. Regardless, that probably doesn't need to block this PR.

Reply:
We discussed this at the meeting, the conclusion was it's fine for the official support to only go through `jquery-wp-docker`.